sdk/servicebus/service-bus/test/internal/utils.spec.ts

```diff
@@ -3,16 +3,89 @@
 
 import {
   checkAndRegisterWithAbortSignal,
-  waitForTimeoutOrAbortOrResolve
+  waitForTimeoutOrAbortOrResolve,
+  getISO8601DurationInSeconds,
+  getISO8601DurationFromSeconds
 } from "../../src/util/utils";
 import { AbortController, AbortError, AbortSignalLike } from "@azure/abort-controller";
 import { delay } from "rhea-promise";
 import chai from "chai";
 import chaiAsPromised from "chai-as-promised";
 chai.use(chaiAsPromised);
 const assert = chai.assert;
+chai.use(chaiAsPromised);
+const should = chai.should();
 
 describe("utils", () => {
+  describe("ISO-8601 Time and duration conversions", () => {
```

Comment:
This is awesome. Can we tack on some invalid cases too?

Reply:
Will add

Reply:
Added!